debug.html

```diff
@@ -268,7 +273,10 @@ <h4>Apply privileges :</h4>
         >
       </p>
     </footer>
+    <script src="/js/utils.js"></script>
```

Comment:
Why this is being added ?

Reply:
These were the necessary files needed for  greeting  to function properly